CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG
 
+## 0.3.0
+
+#### Features
+
+- Added `iosSimulatorArm64`, `watchosSimulatorArm64`, `tvosSimulatorArm64`, `macosArm64` target support ([PR#142](https://github.com/Kotlin/kotlinx-datetime/pull/142)).
```

Comment:
Better to reference the corresponding issue, if there is one.

Reply:
Fixed.